ls1intum/Artemis
# Fix hint and explanation flickering on drag and drop exercises

### Checklist

### Motivation and Context



fixes #3743 

### Steps for Testing


1. Create a drag and drop quiz with a hint and an explanation
2. preview quiz
3. verify that you are able to see the hints/explanations on hover.

### Test Coverage
unchanged

### Screencast




https://user-images.githubusercontent.com/44805696/125445904-1895db4c-23d4-491c-9eb5-7dba184ed123.mp4
